packages/webcert/src/components/fmb/FMBPanelDiagnosisInfo.tsx

```diff
@@ -23,14 +23,19 @@ const EmptyWrapper = styled.div`
 
 interface Props {
   fmbDiagnosisCodeInfo: FMBDiagnosisCodeInfo
+  hasNoFMBSupport: boolean
 }
 
-const FMBPanelDiagnosisInfo: React.FC<Props> = ({ fmbDiagnosisCodeInfo }) => {
+const FMBPanelDiagnosisInfo: React.FC<Props> = ({ fmbDiagnosisCodeInfo, hasNoFMBSupport }) => {
   if (!fmbDiagnosisCodeInfo.diagnosTitle) {
     return (
       <EmptyWrapper className="iu-m-none">
         <InfoBox type={'observe'}>
-          <p>För den angivna diagnosen finns för tillfället inget FMB-stöd.</p>
+          <p>
+            {hasNoFMBSupport
+              ? 'För de angivna diagnoserna finns för tillfället inget FMB-stöd.'
+              : 'För den angivna diagnosen finns för tillfället inget FMB-stöd.'}{' '}
```

Comment:
Should this be here? {' '}

Reply:
I noticed that when formatting with prettier it adds these squiggly brackets with a string inside. Perhaps some rule that needs to be looked at